### PR TITLE
IATIIMPORT-179 Organizations are not being imported

### PR DESCRIPTION
**Pull request by gmutuhu**
IATIIMPORT-179 Organizations are not being imported

### CODE REVIEW

File: import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java [hunk 1]
```diff
@@ -737,20 +737,24 @@ protected void instantiateStaticFields() {
 		Field participatingOrg = new Field("Funding Organization", "participating-org",
                 FieldType.ORGANIZATION,true, getTranslationForField("participating-org"));
 		participatingOrg.setSubType("Funding");
+		participatingOrg.setSubTypeCode("1");
```
Comment (jdeanquin-dg) [inline import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java]: lets try to use constants so its more clear to know that subtype 1 is for participating-org

File: import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java [hunk 2]
```diff
@@ -737,20 +737,24 @@ protected void instantiateStaticFields() {
...
 		getFields().add(participatingOrg);
 		
 		Field accountableOrg = new Field("Accountable Organization", "participating-org",
                 FieldType.ORGANIZATION, true ,getTranslationForField("participating-org"));
+		accountableOrg.setSubTypeCode("2");
```
Comment (jdeanquin-dg) [inline import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java]: idem

File: import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java [hunk 3]
```diff
@@ -737,20 +737,24 @@ protected void instantiateStaticFields() {
...
 		accountableOrg.setSubType("Accountable");
 		getFields().add(accountableOrg);
 
 		Field extendingOrg = new Field("Extending Organization", "participating-org",
                 FieldType.ORGANIZATION, true, getTranslationForField("participating-org"));
+		extendingOrg.setSubTypeCode("3");
```
Comment (jdeanquin-dg) [inline import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java]: idem

File: import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java [hunk 4]
```diff
@@ -737,20 +737,24 @@ protected void instantiateStaticFields() {
...
 		extendingOrg.setSubType("Extending");
 		getFields().add(extendingOrg);
 
 		Field implementingOrg = new Field("Implementing Organization", "participating-org",
                 FieldType.ORGANIZATION, true, getTranslationForField("participating-org"));
+		implementingOrg.setSubTypeCode("4");
```
Comment (jdeanquin-dg) [inline import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/IATI1XProcessor.java]: idem

File: import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/AMPStaticProcessor.java [hunk 5]
```diff
@@ -298,6 +298,13 @@ private void updateProject(JsonBean project, InternalDocument source, List<Field
 			case LIST:
 				processListDestinationProjects(source, valueMappings, project, mapping, sourceField, destinationField);
 				break;
+			 case ORGANIZATION:
```
Comment (jdeanquin-dg) [inline import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/AMPStaticProcessor.java]: lets add a constant for funding organization and provider organization
Also the code here is the same as the one in line 595. Lets please use a function to do the common code

Comment (gmutuhu) [reply import-core/import-services/src/main/java/org/devgateway/importtool/services/processor/AMPStaticProcessor.java]: I have replaced the literal strings with constants.
I have created a separate ticket for some further refactoring: https://jira.developmentgateway.org/browse/IATIIMPORT-201

**Pull request merged**